api-module-library/frontify/api.js

```diff
@@ -343,6 +343,41 @@ class Api extends OAuth2Requester {
         };
     }
 
+    async listCollectionsAssets(query) {
+      const ql = `query ListCollectionsAssetsForLibrary {
+                    library(id: "${query.libraryId}") {
+                      id
+                      name
+                      collections {
+                        items {
+                          id
+                          name
+                          __typename
+                          assets	{
+                              items	{
+                                id
+                              title
+                                description
+                              tags {
+                                source
+                                value
+                              }
+                              __typename
```

Comment:
Perhaps we need to render all properties here too, using `_filesQuery` method? cc @leofmds

Reply:
Ok @roboli, I included ${this._filesQuery()} according to the other queries that contain assets. Thanks.